lib/models/Onepiece.js

```diff
@@ -0,0 +1,55 @@
+const pool = require('../utils/pool');
+
+module.exports = class Character {
+  id;
+  name;
+  crew;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.crew = row.crew;
+  }
+
+  static async insert({ name, crew }) {
+    const { rows } = await pool.query(
+      'INSERT INTO onepiece(name, crew) VALUES ($1, $2) RETURNING *;',
+      [name, crew]
+    );
+    const character = new Character(rows[0]);
```

Comment:
We might add an early return to our static methods here to prevent some errors if no data is returned from a query to the database. Something like `if(!rows[0]) return null`